### PR TITLE
UI Flow Merge

### PR DESCRIPTION
**Pull request by jacobrs**
Merged the register and login flow at the UI level, also added some images for favicon and brand logo. Also implemented a default picture and a message to make the layout more consistent.

### CODE REVIEW

File: client/src/components/Login.vue [hunk 1]
```diff
@@ -2,13 +2,12 @@
...
       <div class="row">
-        <div id="login-header">
-          <button id="login-btn" v-on:click="changeFormType($event)" v-bind:class="{'primary-btn': showingLoginForm, 'secondary-btn': showingRegisterForm}">Login</button>
-          <button id="register-btn" v-on:click="changeFormType($event)" v-bind:class="{'primary-btn': showingRegisterForm, 'secondary-btn': showingLoginForm}">Register</button>
-        </div>
-      </div>
-      <div class="row">
         <div id="login-form" class="column column-50 column-offset-25">
+          <div id="user-info-div" >
+              <span v-show="currentSlackRealname">Welcome, {{currentSlackRealname}}</span>
```
Comment (lancelafontaine) [inline client/src/components/Login.vue:7]: I noticed that some people did not input their names in Slack as it's optional. Only their username is required. In this instance, `currentSlackRealname` would be `""` and this section wouldn't show.

**Review by lancelafontaine — CHANGES_REQUESTED** [review]
Tiny change requested, but otherwise look great!

File: client/src/components/Login.vue (hunk 1 above)
Comment (jacobrs) [reply client/src/components/Login.vue:7]: @lancelafontaine so do you think we should we use the username instead?

Comment (lancelafontaine) [reply client/src/components/Login.vue:7]: Sure. Or maybe default to real name, unless it doesn't exist, then user name? Either should work :)

Comment (jacobrs) [reply client/src/components/Login.vue:7]: Done! :+1:

**Review by lancelafontaine — APPROVED** [review]
Wootwoot! 🚢  it

**Pull request merged**